src/components/panel/panel.spec.js

```diff
@@ -2713,6 +2713,70 @@ describe('$mdPanel', function() {
     });
   });
 
+  describe('contentElement support: ', function() {
+    var config;
+
+    beforeEach(function() {
+      config = {
+        contentElement: angular.element('<div>'),
+        position: $mdPanel.newPanelPosition().absolute().center()
+      };
+    });
+
+    it('should wrap the content element in the proper HTML and assign ' +
+      'the wrapper to the panel reference', function() {
+        openPanel(config);
+
+        expect(panelRef.panelEl.parent()).toHaveClass(PANEL_WRAPPER_CLASS.substring(1));
```

Comment:
I think I would prefer to fix the variables over using substrings like this. It's confusing from the variable name that you're just trying to get rid of the "." What do you think about the following?

The _CLASS should just be the class names, not the selectors. Then the not _CLASS versions should be the selectors. I think that flows better with the expects anyways. ```expect(PANEL_WRAPPER_CLASS).toHaveClass(HIDDEN_CLASS)``` doesn't make sense to me.

So,
```
var PANEL_WRAPPER = ".md-panel-outer-wrapper";
var PANEL_WRAPPER_CLASS = "md-panel-outer-wrapper";
```